lib/daru/io/exporters/rds.rb

```diff
@@ -3,6 +3,8 @@
 module Daru
   module IO
     module Exporters
+      # RDS Exporter Class, that extends +to_rds+ method to +Daru::DataFrame+
+      # instance variables
```

Comment:
I believe it is wise to set YARD to use Markdown markup, and use it in comments.